ssc/cmod_pvsamv1.cpp

```diff
@@ -2630,6 +2637,14 @@ void cm_pvsamv1::exec()
         }
         assign("nominal_annual_clipping_output", nominal_annual_clipping_output);
     }
+    if (as_boolean("enable_subinterval_distribution")) {
```

Comment:
It looks like this functionality could be run at each timestep in the main time loop below, nothing is being pre-calculated, is that correct? If so, avoiding an extra loop would be preferable.

Reply:
Running sharedInverter->calculatePower() after the regular AC calculations was causing issues with the calculations, so I was calculating the max AC power needed ahead of time. There might be a workaround solution but it would require more thinking on the architecture.